aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/s3/S3ExpressDecorator.kt

```diff
@@ -83,25 +80,6 @@ class S3ExpressDecorator : ClientCodegenDecorator {
             S3ExpressRequestChecksumCustomization(
                 codegenContext, operation,
             )
-
-    override fun extraSections(codegenContext: ClientCodegenContext): List<AdHocCustomization> {
-        return listOf(
-            adhocCustomization<SdkConfigSection.CopySdkConfigToClientConfig> { section ->
-                rust(
-                    """
-                    ${section.serviceConfigBuilder}.set_disable_s3_express_session_auth(
-                        ${section.sdkConfig}
-                            .service_config()
-                            .and_then(|conf| {
-                                let str_config = conf.load_config(service_config_key("AWS_S3_DISABLE_EXPRESS_SESSION_AUTH", "s3_disable_express_session_auth"));
```

Comment:
Do we want to disable these changes that have no global value? It seems like we could be strategic and only disable the region one. I'm fine with the larger revert though

Reply:
Good point, I'll leave this one in.